x/mint/keeper/genesis.go

```diff
@@ -0,0 +1,31 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v10/x/mint/types"
+)
+
+// InitGenesis new mint genesis.
```

Comment:
```suggestion
// InitGenesis checks if the ModuleAddress is created to and returns the last reduction epoch number. 
```

Maybe this could be elaborated